ddtrace/contrib/azure_functions/patch.py

```diff
@@ -0,0 +1,14 @@
+from ddtrace.contrib.internal.azure_functions.patch import *  # noqa: F403
+from ddtrace.internal.utils.deprecations import DDTraceDeprecationWarning
+from ddtrace.vendor.debtcollector import deprecate
+
+
+def __getattr__(name):
```

Comment:
There's no need to deprecate these newly-added functions.

Reply:
All of the other `contrib/*/patch.py` files use this deprecate method so I copy/pasted it. For my own understanding, why is this code included in the other integrations if they're also not deprecated?